ta4j-core/src/main/java/org/ta4j/core/analysis/criteria/NumberOfBreakEvenTradesCriterion.java

```diff
@@ -0,0 +1,38 @@
+package org.ta4j.core.analysis.criteria;
+
+import org.ta4j.core.TimeSeries;
+import org.ta4j.core.Trade;
+import org.ta4j.core.TradingRecord;
+import org.ta4j.core.num.Num;
+
+/**
+ * Number of break even trades criterion.
+ */
+public class NumberOfBreakEvenTradesCriterion extends AbstractAnalysisCriterion {
+
+    @Override
+    public Num calculate(TimeSeries series, TradingRecord tradingRecord) {
+        long numberOfLosingTrades = tradingRecord.getTrades().stream()
+                .filter(trade -> trade.isClosed())
```

Comment:
Lambda can be replaced with method reference: filter(Trade::isClosed)

Reply:
skip